translations/en.json

```diff
@@ -1,6 +1,7 @@
 {
 	"LOADING": "Loading …",
 
+	"YESTERDAY": "Yesterday",
```

Comment:
So, what are english displays showing then?

Reply:
They will show "Yesterday" like "Today" for today

Reply:
I added some checks to prove that the CSS classes got added correctly which was more or less copy and paste. I will try to add more changes but i will need to dive deeper into into the test classes to check the locals